#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

Header header
PodHeader pod_header
bool switch_forward
bool switch_reverse
bool switch_neutral
bool forward_selected
bool reverse_selected
bool neutral_selected
uint8 derating
uint8 vehicle_speed
uint8 conf_speed_limit
uint8 throttle_ch1
uint8 throttle_ch2
